src/user_interface.py

```diff
@@ -609,7 +617,7 @@ def __confirm(self):
             self.__ready_to_continue.set(1)
 
     def __show_help(self):
-        helpfile = str(Path(__file__).parent / "assets" / "manual.pdf")
+        helpfile = str(Path(__file__).parent / "assets" / "manual.md")
```

Comment:
The PDF is intentionally not included in the source code; it's only included in the built output. This is a step in the build instructions:

https://github.com/iansan5653/open-mcr/blob/ce8cd14b41aadd5ca3aa939706ec70b3c59c14f0/build_instructions.md?plain=1#L9-L10

I would rather not link to the Markdown file because it's not very readable.

I do understand that this makes the help button broken if you are running from the plain source code. Maybe a better solution would be to build the PDF on startup if it doesn't already exist. Or maybe we could use HTML instead of PDF since HTML is easier to build from Markdown. But let's take this change out of this pull request and we can discuss further in a new issue.

Reply:
reverted in https://github.com/iansan5653/open-mcr/pull/74/commits/ec27ba7b8933ec6bb6da8217ad7b960aa566637f